pom.xml

```diff
@@ -1470,5 +1470,31 @@
       </build>
     </profile>
 
+    <profile>
+      <id>pitest-size</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.pitest</groupId>
+            <artifactId>pitest-maven</artifactId>
+            <version>1.1.10</version>
+            <configuration>
+              <targetClasses>
+                <param>com.puppycrawl.tools.checkstyle.checks.sizes.*</param>
+              </targetClasses>
+              <targetTests>
+                <param>com.puppycrawl.tools.checkstyle.checks.sizes.*</param>
+              </targetTests>
+              <mutationThreshold>88</mutationThreshold>
+              <timeoutFactor>10</timeoutFactor>
+              <timeoutConstant>50000</timeoutConstant>
+              <failWhenNoMutations>false</failWhenNoMutations>
```

Comment:
`failWhenNoMutations: Whether to throw error when no mutations found.`

Why don't we set this to true?